### ISSUE TITLE
Unsupported OpenType signature ttcf after changing to Noto Sans CJK TC

### ISSUE THREAD

**Issue by GreatHaop** [sedwards2009/extraterm#404]
```

/home/user/extratermqt-0.68.0-linux-x64/main/dist/main.cjs:13537
          throw new Error("Unsupported OpenType signature " + signature);
                ^
Error: Unsupported OpenType signature ttcf
    at parseBuffer (/home/user/extratermqt-0.68.0-linux-x64/main/dist/main.cjs:13537:17)
    at /home/user/extratermqt-0.68.0-linux-x64/main/dist/main.cjs:13704:22
    at /home/user/extratermqt-0.68.0-linux-x64/main/dist/main.cjs:13437:11
    at FSReqCallback.readFileAfterClose [as oncomplete] (node:internal/fs/read_file_context:68:3)
panic: Post "/command": unsupported protocol scheme ""

goroutine 1 [running]:
main.runCommand({0x0, 0x0}, 0xc0000bbf08)
        /home/runner/work/extraterm-launcher/extraterm-launcher/cmd/extraterm-launcher/extraterm-launcher.go:200 +0x2f4
main.runOpenWindowAtCommand({0x0, 0x0}, {0xc00001a004, 0x27})
        /home/runner/work/extraterm-launcher/extraterm-launcher/cmd/extraterm-launcher/extraterm-launcher.go:135 +0xd9
main.main()
        /home/runner/work/extraterm-launcher/extraterm-launcher/cmd/extraterm-launcher/extraterm-launcher.go:40 +0x73
```

**Comment by sedwards2009** [comment]
This sound like issue #402 . It might already be fixed now since the upgrade to Qt6. Could you retest this issue with the latest Extraterm version?

**Comment by sedwards2009** [comment]
Fixed in 0.71.1.
